robyn/responses.pyi

```diff
@@ -0,0 +1,19 @@
+from typing import Any
+
+def static_file(file_path: str) -> dict[str, Any]:
+    """
+    This function will help in serving a static_file
+
+    :param file_path str: file path to serve as a response
+    """
+
+    pass
+
+def jsonify(input_dict: dict) -> str:
```

Comment:
Even if the type arguments for a generic type don't matter in that context you should still use `Any` or `object` in its place. This is because in strict mode, pyright will report an error if a type annotation is unknown (i.e. missing type arguments).

Reply:
Thank you @RobertCraigie! I will include it in the next release! :D